service/collector/app.go

```diff
@@ -154,10 +154,13 @@ func (a *App) collectAppStatus(ctx context.Context, ch chan<- prometheus.Metric)
 			team = formatTeamName(key.AppTeam(app))
 		}
 
+		// Trim `v` prefix from App CR version if there is any
```

Comment:
Can you add a TODO comment that we want to remove this once flux supports removing the prefix?